lib/api/CDataFrameTrainBoostedTreeClassifierRunner.cc

```diff
@@ -162,9 +162,9 @@ void CDataFrameTrainBoostedTreeClassifierRunner::writeOneRow(
     }
 
     if (featureImportance != nullptr) {
+        int numberClasses{static_cast<int>(classValues.size())};
```

Comment:
Historically we always used `size_t` to iterate container indices.  Why are we switching to `int`?

Reply:
Eigen's index type is unsigned, so we use `int` when we're iterating over components of an Eigen vector.